### ISSUE TITLE
Time and Space Complexity

### ISSUE THREAD

**Issue by zx19950617** [ethnhe/PVN3D#10]
Thanks for your excellent work and sharing! 
Could you summarize space and time complexity of PVN3D, please? PVN3D seems to achieve real time performance, right? Any quantitative metric?

Besides, my training procedure is slightly inefficient. As exmaple, the training of --cls ape is conducted on two Tesla t4 GPUs, each with 16 G memory. Detailed training paremeters remain unchanged except the self.mini_batch_size and self.val_mini_batch_size (default 24 to 8).
Unacceptably, the total training time may take around 10 days for just the class ape. So I'd like to confirm if any problems occured in my training stage.
![图片](https://user-images.githubusercontent.com/38937499/78877816-02d4e580-7a84-11ea-887b-57e1cf43ed39.png)
![图片](https://user-images.githubusercontent.com/38937499/78877824-04061280-7a84-11ea-8e92-ffa36de561c9.png)
![图片](https://user-images.githubusercontent.com/38937499/78877834-08323000-7a84-11ea-9c23-ac1d09465998.png)

Thanks again! And great appreciation for any help!

**Comment by ethnhe** [comment]
During inference, it takes 0.17 seconds for network forward propagation, most of which spent on PointNet++, and 0.02 seconds for pose estimation of each object on an RTX 2080 Ti. The overall runtime is 5 FPS on the LineMOD dataset.  It takes about 36 hours training the total 25 epochs with 8 RTX 2080 Ti. But you don't need to finish all the 25 epochs, it usually get a good result with about 9~10 epochs and best result with about 18~20 epochs.

**Comment by zx19950617** [comment]
OK, thanks for your experience sharing and suggestion!  
I got a general understanding about the training and inference of PVN3D.
